services/manifest-repo-export-service/pkg/cmd/server.go

```diff
@@ -303,6 +308,12 @@ func Run(ctx context.Context) error {
 		logger.FromContext(ctx).Sugar().Infof("Custom Migrations skipped. Kuberpult only runs custom Migrations if" +
 			"KUBERPULT_MINIMIZE_EXPORTED_DATA=false and KUBERPULT_CHECK_CUSTOM_MIGRATIONS=true.")
 	}
+	if dbGitTimestampMigrationEnabled {
+		err := dbHandler.RunCustomMigrationReleasesTimestamp(ctx, repo.State().GetAppsAndTeams, repo.State().FixReleasesTimestamp)
+		if err != nil {
+			return fmt.Errorf("error running migrations for fixing releases timestamp")
```

Comment:
do not leave errors behind please
```suggestion
			return fmt.Errorf("error running migrations for fixing releases timestamp: %w", err)
```